cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/cube/CubeDeleteQuery.java

```diff
@@ -40,14 +42,14 @@
    * @param endTs end time of the data selection, in seconds since epoch
    * @param resolution resolution of the aggregations to delete from
    * @param sliceByTagValues tag name, tag value pairs that define the data selection
-   * @param measureName name of the measure to delete, {@code null} means delete all
+   * @param measureNames name of the measures to delete, {@code null} means delete all
```

Comment:
alright - as I went thru rest of code:
* measureNames field should never be null
* do not allow null as value for param for ctor that takes in measureNames
* add ctor that takes no measureNames param - which means "all metrics"